addons/point_of_sale/models/pos_session.py

```diff
@@ -345,6 +351,20 @@ def open_cashbox(self):
 
         return action
 
+    @api.multi
```

Comment:
`@api.multi` is the default mode, no need to specify it (same for the method above as well)